### PR TITLE
Tutorial: Build an NFT App powered by FuseBox.

### PR DESCRIPTION
**Pull request by emmaodia**

### CODE REVIEW

File: docs/tutorials/tutorials/mint-erc721-aa.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+```javascript
+// hardhat.config.js
+require("@nomiclabs/hardhat-waffle");
+
+module.exports = {
+  solidity: "0.8.0",
+  networks: {
+    fuse: {
+      url: `https://eth-kovan.alchemyapi.io/v2/${ALCHEMY_KEY}`,
```
Comment (LiorAgnin) [inline docs/tutorials/tutorials/mint-erc721-aa.mdx:104]: @emmaodia Shouldn't the URL be https://rpc.fuse.io?

Comment (emmaodia) [reply docs/tutorials/tutorials/mint-erc721-aa.mdx:104]: Fixed.

File: docs/tutorials/tutorials/mint-erc721-aa-part2.mdx [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+```
+
+Answer the required prompts from NextJS in the terminal. It is important to note that we use TypeScript for this application and Tailwind CSS.
+
+## Step 3: Install Dependencies
+
+Install the required dependencies:
+
+```bash
+npm install @fuseio/fusebox-web-sdk
```
Comment (LiorAgnin) [inline docs/tutorials/tutorials/mint-erc721-aa-part2.mdx:48]: @emmaodia You forgot to `ethers` here.

**Review by LiorAgnin — APPROVED** [review]
LGTM!

**Pull request merged**